### ISSUE TITLE
PIP package incomplete

### ISSUE THREAD

**Issue by felixtech-msp** [uilianries/conan-ldap-authentication#22]
Referring to issue #20
I was unable to re-open this issue as it was closed by a collaborator and therefore cannot be reopened by the issuer.

The pip package for version 0.4.0 has a missing file.

```
Collecting conan_ldap_authentication
  Downloading https://files.pythonhosted.org/packages/74/cf/4af6ac2d036d65ebfe4481198089414473f4b01f4fa2527b52069a3563f1/conan_ldap_authentication-0.4.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-w65df3tn/conan-ldap-authentication/setup.py", line 131, in <module>
        install_requires=Requirements.get(),
      File "/tmp/pip-install-w65df3tn/conan-ldap-authentication/setup.py", line 34, in get
        with open(Requirements.__requirements_path) as req_file:
      File "/usr/local/lib/python3.7/codecs.py", line 898, in open
        file = builtins.open(filename, mode, buffering)
    FileNotFoundError: [Errno 2] No such file or directory: 'conan_ldap/requirements.txt'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-w65df3tn/conan-ldap-authentication/
You are using pip version 19.0.1, however version 22.0.4 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
The command '/bin/sh -c pip install conan_ldap_authentication' returned a non-zero code: 1
```

**Comment by uilianries** [comment]
Thank you reporting, I'll check asap.

**Comment by uilianries** [comment]
@felixtech-msp I just released the version 0.4.1 which should fix it. Please, update.
